registry/handlers/referrers.go

```diff
@@ -59,13 +64,25 @@ func (h *referrersHandler) GetReferrers(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	// extract the artifactType filter.
 	var annotations map[string]string
 	var artifactTypeFilter string
 	if artifactTypeFilter = r.URL.Query().Get("artifactType"); artifactTypeFilter != "" {
 		annotations = map[string]string{
 			v1.AnnotationReferrersFiltersApplied: "artifactType",
 		}
 	}
+
+	// extract the page size info. Users define page size by using the n
+	// query parameter. The default page size is 100.
+	pageSize, nParseError := strconv.Atoi(r.URL.Query().Get("n"))
+	if nParseError != nil || pageSize < minPageSize || pageSize > maxPageSize {
+		pageSize = maxPageSize
+	}
+
+	// extract the page number info.
+	pageNumber, _ := strconv.Atoi(r.URL.Query().Get("p"))
+
 	referrers, err := h.generateReferrersList(h, h.Digest, artifactTypeFilter)
```

Comment:
Although it is not efficient, it is still good for testing purpose. Could you document that the pagination is simulated for testing purpose?

Reply:
Added the doc comment.